contracts/src/Zappers/GasCompZapper.sol

```diff
@@ -275,6 +281,8 @@ contract GasCompZapper is BaseZapper {
         require(msg.sender == address(flashLoanProvider), "GCZ: Caller not FlashLoan provider");
 
         LatestTroveData memory trove = troveManager.getLatestTroveData(_params.troveId);
+        uint256 collLeft = trove.entireColl - _params.flashLoanAmount;
```

Comment:
In the WETHZapper we account for `ETH_GAS_COMPENSATION` too. Is intentionally left out here (if, so why)?

Reply:
In case of WETH, the collateral and gas compensation tokens are the same, so we can send it to the user in one combined transfer (of raw ETH, following a withdrawal from WETH), hence the addition of `ETH_GAS_COMPENSATION`.

Reply:
Of course, we could have kept `ETH_GAS_COMPENSATION` out of `collLeft` and dealt with it later, when doing the withdrawal & transfer.

The downside of the current implementation is that the frontend has to calculate `minExpectedCollateral` differently between (W)ETH and other collateral branches.

Reply:
Yes, because gas compensation is WETH, while here coll will be another token (wstETH or RETH).